addons/website_slides/controllers/main.py

```diff
@@ -1231,6 +1231,8 @@ def _slide_embed(self, slide_id, page="1", is_external_embed=False, **kw):
                 raise werkzeug.exceptions.NotFound()
 
             referer_url = request.httprequest.headers.get('Referer', '')
+            if not slide.active:
```

Comment:
Could be concatenated with the above condition already raising a NotFound

Reply:
Done :+1: